ci-operator/step-registry/hypershift/mce/aws/create/hypershift-mce-aws-create-chain.yaml

```diff
@@ -4,6 +4,9 @@ chain:
     - as: create-hostedcluster
       cli: latest
       env:
+        - name: MCE_VERSION
+          default: "2.2"
```

Comment:
should default be latest 2.4?

Reply:
https://issues.redhat.com/browse/OCPBUGS-17673, So we don't set default to `2.4`